remill/Arch/X86/Semantics/X87.cpp

```diff
@@ -71,6 +71,41 @@ namespace {
 #define DEF_FPU_SEM(name, ...) \
     DEF_SEM(name, ##__VA_ARGS__, PC pc, I16 fop)
 
+// TODO(joe): Loss of precision, see issue #199.
+DEF_FPU_SEM(FBLD, RF80W, MD80 src1) {
+  SetFPUIpOp();
+  SetFPUDp(src1);
+
+  auto src1_dec = ReadDec80(src1);
+  auto val = 0.0;  // Decoded BCD value
```

Comment:
Perhaps be explicit for `val` and `mag` being `double`s. I know that floating point constants are defaulted to that type, but in this case, it's nice to be explicit.